gateway.go

```diff
@@ -132,5 +134,11 @@ func (s *gatewayResource) configHandler(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
+	// Make expiration time even/random throughout interval 12-36h
+	// So the avg will be 24h but without spikes of the requests for the key renewal
+	rand.Seed(time.Now().UnixNano())
+	maxAge := 12*3600 + rand.Intn(24*3600)
```

Comment:
Can we make these constants so we can reference them from the tests, too?

Reply:
sure, adressed